ui/src/app/kvm/components/CertificateMetadata/_index.scss

```diff
@@ -0,0 +1,14 @@
+@mixin CertificateMetadata {
+  .certificate-metadata {
+    @extend %vf-card;
+    @extend %vf-is-bordered;
+  }
+
+  .certificate-metadata__list {
+    margin-bottom: 0;
```

Comment:
Could this have used a `u-no-margin--bottom` instead?

Reply:
Yep! done